Cargo.toml

```diff
@@ -191,11 +190,190 @@ opt-level = 1
+regex = "1.7.1"
+reqwest = { version = "0.11.18", default_features= false, features = ["blocking", "json", "rustls-tls"] }
+roaring = "0.10.1"
+rocksdb = { version = "0.21.0", features = ["snappy", "lz4", "zstd", "zlib", "multi-threaded-cf"], default-features = false }
+ron = "0.8.0"
+rstest = "0.16.0"
+rusoto_core = { version = "0.48.0", default_features=false, features=["rustls"] }
+rusoto_kms = { version = "0.48.0", default_features=false, features=["rustls"] }
+rust-version = "1.56.1"
+rustls = { version = "0.20.4", features = ["dangerous_configuration"] }
+rustls-pemfile = "1.0.2"
+rustversion = "1.0.9"
+rustyline = "9.1.2"
+rustyline-derive = "0.7.0"
+schemars = { version = "0.8.10", features = ["either"] }
+scopeguard = "1.1"
+serde = { version = "1.0.144", features = ["derive", "rc"] }
+serde-name = "0.2.1"
+serde-reflection = "0.3.6"
+serde_json = "1.0.95"
+serde_repr = "0.1"
+serde_test = "1.0.147"
+serde_with = { version = "2.1.0", features = ["hex"] }
+serde_yaml = "0.9.21"
+shell-words = "1.1.0"
+shellexpand = "3.1.0"
+signature = "1.6.0"
+similar = "2.2.0"
+slip10_ed25519 = "0.1.3"
+smallvec = "1.10.0"
+snap = "1.1.0"
+static_assertions = "1.1.0"
+structopt = "0.3.26"
+strum = { version = "0.24", features = ["derive"] }
+strum_macros = "0.24.3"
+syn = { version = "1.0.104", features = ["full", "derive", "extra-traits"] }
+# syn = { version = "2", features = ["full", "fold", "extra-traits"] }
+synstructure = "0.12"
+sysinfo = "0.27.5"
+tap = "1.0.1"
+tempfile = "3.3.0"
+test-fuzz = "3.0.4"
+thiserror = "1.0.40"
+tiny-bip39 = "1.0.0"
 tokio = "1.28.1"
-axum = { version = "0.6.6", default-features = false, features = ["headers", "tokio", "http1", "http2", "json", "matched-path", "original-uri", "form", "query"] }
+tokio-retry = "0.3"
+tokio-rustls = "0.23"
+tokio-stream = { version = "0.1.14", features = ["sync", "net"] }
+tokio-util = "0.7.4"
+toml = { version = "0.7.4", features = ["preserve_order"] }
+toml_edit = { version = "0.19.10" }
+tonic = { version = "0.8.2", features = ["transport", "tls"] }
+tonic-build = { version = "0.8.2", features = [ "prost", "transport" ] }
+tonic-health = "0.8.0"
+tower = { version = "0.4.12", features = ["full", "util", "timeout", "load-shed", "limit"] }
+tower-http = { version = "0.3.4", features = ["cors", "full", "trace", "set-header", "propagate-header"] }
+# tower-http = { version="0.4", features = ["trace"] }
+tower-layer = "0.3.2"
+tracing = "0.1.37"
+tracing-appender = "0.2.2"
+tracing-subscriber = { version = "0.3.15", default-features = false, features = ["std", "smallvec", "fmt", "ansi", "time", "json", "registry", "env-filter"] }
+ttl_cache = "0.5.1"
+uint = "0.9.4"
+unescape = "0.1.0"
+url = "2.3.1"
+uuid = { version = "1.1.2", features = ["v4", "fast-rng"]}
+webpki = { version = "0.22.0", features = ["alloc", "std"] }
+x509-parser = "0.14.0"
+zstd = "0.12.3"
+versions = "4.1.0"
 
 # Move dependencies
+move-bytecode-verifier = { path = "external-crates/move/move-bytecode-verifier" }
```

Comment:
These should not be workspace dependencies, they are part of the execution layer, and the `sui-execution` crate will multiplex between this and other versions of this crate -- I want to avoid other crates depending on these crates directly (there are some crates that are allowed to do that, but they should be the exception, and not the rule).

Reply:
Making them "workspace dependencies" would not impact this at all, afaik